packages/shared/src/dropdown/multiselectdropdown.tsx

```diff
@@ -43,17 +49,17 @@ export const MultiSelectDropdown: React.FC<MultiSelectDropdownProps> = ({
       aria-labelledby={id}
       noResultsFoundText={t('No results found')}
       isCheckboxSelectionBadgeHidden
+      {...rest}
     >
       {items}
     </Select>
   );
 };
 
-export type MultiSelectDropdownProps = {
+export type MultiSelectDropdownProps = SelectProps & {
```

Comment:
selectProps selections, placeholder text, variant, has no effect due to selected, options, and variant.  shall we remove these custom props?

just asking for your opinion 
```suggestion
 type Props = Omit<SelectProps, 'placeholderText' | 'selections'>
export type MultiSelectDropdownProps = Props & {
```